listener/controller/apibinding_controller_test.go

```diff
@@ -68,8 +70,12 @@ func TestAPIBindingReconciler_Reconcile(t *testing.T) {
 				tt.mockSetup(ioHandler, discoveryFactory, apiSchemaResolver, clusterPathResolver)
 			}
 
-			r := controller.NewAPIBindingReconciler(ioHandler, discoveryFactory, apiSchemaResolver, clusterPathResolver)
-			_, err := r.Reconcile(context.Background(), ctrl.Request{ClusterName: tt.clusterName})
+			loggerCfg := logger.DefaultConfig()
```

Comment:
since we are touching this code, lets move it out of the `for _, tt := range tests {` loop and initiate one time.

Also, we have TestLogger, maybe it makes sense to start using it in this repo:
https://github.com/openmfp/golang-commons/blob/main/logger/testlogger/test_logger.go#L22
No need to test log output, you can use it with HideLogOutput().